diff --git a/node_modules/vue/dist/vue.runtime.esm.js b/node_modules/vue/dist/vue.runtime.esm.js
index ae5d8a2..63a9223 100644
--- a/node_modules/vue/dist/vue.runtime.esm.js
+++ b/node_modules/vue/dist/vue.runtime.esm.js
@@ -7066,7 +7066,7 @@ function createPatchFunction(backend) {
             }
         }
         else if (elm.data !== vnode.text) {
-            elm.data = vnode.text;
+            // elm.data = vnode.text;
         }
         return true;
     }
@@ -7559,7 +7559,7 @@ function updateDOMProps(oldVnode, vnode) {
             // #6601 work around Chrome version <= 55 bug where single textNode
             // replaced by innerHTML/textContent retains its parentNode property
             if (elm.childNodes.length === 1) {
-                elm.removeChild(elm.childNodes[0]);
+                // elm.removeChild(elm.childNodes[0]);
             }
         }
         if (key === 'value' && elm.tagName !== 'PROGRESS') {
@@ -7595,7 +7595,7 @@ function updateDOMProps(oldVnode, vnode) {
             // some property updates can throw
             // e.g. `value` on <progress> w/ non-finite value
             try {
-                elm[key] = cur;
+                // elm[key] = cur;
             }
             catch (e) { }
         }
